experiments/golden-results/UKNI-Information-Barrier-summary.txt

```diff
@@ -98,6 +73,11 @@ Redacted compiler error message:
 unmatched actual "REDACTED" in call
 Raw compiler error message:
 --
+Occurs: 3 times
```

Comment:
Here we might be overly cautious as to what `type` of symbol we allow to form the range-bound.